CHANGELOG.md

```diff
@@ -22,6 +22,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 - Vi mode keybinding (z) to center view around vi mode cursor
 - Accept hexadecimal values starting with `0x` for `--embed`
 - Config option `cursor.blink_timeout` to timeout cursor blinking after inactivity
+- On macOS, use the `AppleFontSmoothing` user default to decide whether fonts should be "smoothed"
```

Comment:
I'd probably put it into changed in the following wording:

"On macOS `AppleFontSmoothing` user default is now being used to control font smoothing"

Reply:
Reworded.